library/std/src/sys/pal/windows/fs.rs

```diff
@@ -1132,16 +1186,16 @@ pub fn rmdir(p: &Path) -> io::Result<()> {
 }
```

Comment:
Shouldn't all the methods above (`rename`, `unlink`, `rmdir`, etc) also be switched to `NativePath`?